src/main/java/salesforce/auth/Authentication.java

```diff
@@ -5,11 +5,13 @@
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
  */
-package auth;
+package salesforce.auth;
 
-import config.*;
+import org.apache.http.HttpHeaders;
+import salesforce.config.*;
+import salesforce.entities.Token;
 
-import static config.EnvConfigurationFile.dotenv;
+import static salesforce.config.EnvConfigurationFile.dotenv;
 import static io.restassured.RestAssured.given;
 
 public class Authentication {
```

Comment:
Could be generalized so it can perform different types of authentication rather than only supporting defined parameters in the header from a file.